tests/common/common_vec.h

```diff
@@ -72,9 +72,9 @@ bool check_vector_values(sycl::vec<vecType, numOfElems> vector,
  *        for division result are accurate enough
  */
 template <typename vecType, int numOfElems>
-typename std::enable_if<is_sycl_floating_point<vecType>::value, bool>::type
-check_vector_values_div(sycl::vec<vecType, numOfElems> vector,
-                        vecType *vals) {
+typename std::enable_if<is_sycl_scalar_floating_point<vecType>::value,
```

Comment:
I would change `enable_if` to `enable_if_t` and `is_sycl_scalar_floating_point` to `is_sycl_scalar_floating_point_v` in this file.

Reply:
Done.